From fb113e3d5d5db12347c524c24d44f2ce5e1470ad Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Fri, 6 Jan 2017 10:58:33 +0800
Subject: [PATCH 1459/5242] MLK-13638-2 ARM: dts: imx7ulp-evk: fix GPIO
 direction for USB vbus and id

commit  7124ce52155ddc95484320ab55fa6397c4aaf20f from
https://source.codeaurora.org/external/imx/linux-imx.git

The vbus should be output, and the id should be input.
Without this change, the GPIO configuration (through pinctrl
register) is incorrect from system suspend.

Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk.dts |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx7ulp-evk.dts b/arch/arm/boot/dts/imx7ulp-evk.dts
index 87857f2..2cd0524 100644
--- a/arch/arm/boot/dts/imx7ulp-evk.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk.dts
@@ -263,13 +263,13 @@
 
 		pinctrl_usb_otg1: usbotg1grp {
 			fsl,pins = <
-				IMX7ULP_PAD_PTC0__PTC0		0x30100
+				IMX7ULP_PAD_PTC0__PTC0		0x20100
 			>;
 		};
 
 		pinctrl_extcon_usb1: extcon1grp {
 			fsl,pins = <
-				IMX7ULP_PAD_PTC8__PTC8		0x30103
+				IMX7ULP_PAD_PTC8__PTC8		0x10103
 			>;
 		};
 
-- 
1.7.9.5

